app/src/main/java/org/wikipedia/travel/trip/Trip.java

```diff
@@ -17,16 +18,24 @@
 //and a departure date
 public class Trip {
 
+    //Initialization of database table for trips with single destinations
+    public static final TripDatabaseTable DATABASE_TABLE = new TripDatabaseTable();
+    //Initialization of database table for destinations with single destinations
+    public static final DestinationDatabaseTable DESTINATION_DATABASE_TABLE = new DestinationDatabaseTable();
     //trip id for database storage
     private long id;
     //trip title, also to be used to signfiy the germane list in the database
-    @NonNull private String title;
-    @SuppressWarnings("unused") @Nullable private List<Destination> destinations;
-    @SuppressWarnings("unused") @Nullable private Destination singleDestination;
-    @SuppressWarnings("unused") @Nullable private Date departureDate;
-
-    //Initialization of database table for trips with single destinations
-    public static final TripDatabaseTable DATABASE_TABLE = new TripDatabaseTable();
+    @NonNull
+    private String title;
+    @SuppressWarnings("unused")
```

Comment:
If these attributes are never used, is it possible to delete them?

Reply:
The attributes existed in the previous merge, it was moved to a new line. With that said, those tags are used in the code.